crates/apub/src/activities/community/report.rs

```diff
@@ -49,8 +50,8 @@ impl Report {
       id: id.clone(),
       audience: Some(community.id().into()),
     };
-
-    let inbox = vec![community.shared_inbox_or_inbox()];
+    // todo: this should probably filter and only send if the community is remote?
+    let inbox = ActivitySendTargets::to_inbox(community.shared_inbox_or_inbox());
```

Comment:
Not necessary to check here, the library already checks and doesnt send to local instance.

Reply:
That is true and right now it doesn't matter too much. But: If in the future we add an optimization to only fetch those activities from the database that actually need to be sent (either at all or to a specific inbox), then it would be useful if this filtering happened before insertion into the DB.

I've removed the comment and replaced it with a filter akin to what is already done in `send_activity_in_community`